web/src/server/auth.ts

```diff
@@ -264,6 +280,26 @@ const extendedPrismaAdapter: Adapter = {
 
     return user;
   },
+
+  async linkAccount(data) {
+    if (!prismaAdapter.linkAccount)
+      throw new Error("linkAccount not implemented");
+
+    if (
```

Comment:
can we remove this here as it is checked in the nextauth callbacks outside of the prisma adapter?

Reply:
Yes, we can, since it's already checked in the callback.